contentctl/objects/config.py

```diff
@@ -249,11 +254,52 @@ class StackType(StrEnum):
     classic = auto()
     victoria = auto()
 
+
 class inspect(build):
-    splunk_api_username: str = Field(description="Splunk API username used for running appinspect.")
-    splunk_api_password: str = Field(exclude=True, description="Splunk API password used for running appinspect.")
+    # TODO (cmcginley): can we enable enrichments by default for the inspect command?
+    splunk_api_username: str = Field(
+        description="Splunk API username used for appinspect and Splunkbase downloads."
+    )
+    splunk_api_password: str = Field(
+        exclude=True,
+        description="Splunk API password used for appinspect and Splunkbase downloads."
+    )
+    # TODO (cmcginley): instead of providing a path to the previous build, we will use the build
+    #   generated as part of inspect; unsure if this is ideal, we should be building once and then
+    #   validating, testing, publishing etc.
+    previous_build: str | None = Field(
+        default=None,
+        description=(
+            "Local path to the previous ESCU build for versioning enforcement (defaults to the "
+            "latest release published on Splunkbase)."
+        )
+    )
     stack_type: StackType = Field(description="The type of your Splunk Cloud Stack")
 
+    def get_previous_package_file_path(self) -> pathlib.Path:
```

Comment:
I used snake-casing for this function name, as that's the standard for Python functions and attributes. I know most methods in the config class and subclasses use camel-casing.

I'd vote to ultimately convert everything to snake-casing, but if you disagree I can make this camel-cased.